lib/common/test/log.go

```diff
@@ -12,7 +12,7 @@ func LogHandler() logging.Handler {
 			return logging.DiscardHandler()
 		},
 		"stdout": func() logging.Handler {
-			return logging.StdoutHandler
+			return logging.CallerStackHandler("%+v", logging.StdoutHandler)
```

Comment:
Maybe just leave it as is ? If it fails, we get the stack trace anyway.

Reply:
What kinds of fails? :) CallerStackHandler uses stacktrace maybe?

Reply:
> What kinds of fails?

When a test fail for example

Reply:
Yep! Test fails print test file line numbers and file name :)